ksql-functional-tests/src/test/resources/query-validation-tests/delimited-value-format.json

```diff
@@ -0,0 +1,222 @@
+{
+  "comments": [
+    "When using value_format DELIMITED, we can define VALUE_DELIMITER as custom character."
+  ],
+  "tests": [
+    {
+      "name": "select delimited value_format",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, NAME varchar, VALUE integer) WITH (kafka_topic='test_topic', value_format='DELIMITED');",
+        "CREATE STREAM S2 as SELECT id, name, value FROM test;"
+      ],
+      "topics": [
```

Comment:
topics section is not required.  This is only required if you want to set up the schema of the topic I believe.

Reply:
So.. all tests except for `validate value_delimiter to be single character` pass just fine without the `topics` block if I pull down the code.

The fact that `validate value_delimiter to be single character` requires this `topics` block is an artefact of the fact the code allows the `CreateSourceProperties` to be created in an invalid state. 

If you fix `CreateSourceProperties` so that is is not created in an invalid state, i.e. by validating the delimiter string via a validator in the `ConfigDef`, then this issue should go away